src/logic/operate.js

```diff
@@ -1,20 +1,31 @@
-// eslint-disable-next-line
 import Big from 'big.js';
 
 const operate = (numberOne, numberTwo, operation) => {
   switch (operation) {
-    case '-':
-      return Number(Big(numberOne)) - Number(Big(numberTwo));
     case '+':
-      return Number(Big(numberOne)) + Number(Big(numberTwo));
+      return Big(numberOne)
+        .plus(numberTwo)
+        .toString();
+
+    case '-':
+      return Big(numberOne)
+        .minus(numberTwo)
+        .toString();
+
+    case 'x':
+      return Big(numberOne)
+        .times(numberTwo || (operation === 'x' ? '1' : '0'))
+        .toString();
     case '/':
-      return Number(Big(numberOne)) / Number(Big(numberTwo));
-    case '/0':
-      return '0';
-    case '%':
-      return Number(Big(numberOne)) % Number(Big(numberTwo));
+      if (numberTwo === '0') {
+        return 'divide by 0 err';
```

Comment:
Handling division by `0` this way may introduce bugs since we also didn't handle invalid inputs. The illustrations below is trying to operate `'divide by 0 err'` as an operand which results in an error sine this string is an `Invalid number`. Let's handle this:

![bolaerror](https://user-images.githubusercontent.com/36057474/72733336-eff2f900-3b97-11ea-9f40-7e05eda32bba.gif)